api/v1alpha1/observatorium_types.go

```diff
@@ -23,17 +23,17 @@ import (
 // ObservatoriumSpec defines the desired state of Observatorium
 type ObservatoriumSpec struct {
 	// Thanos Image name
-	Image string `json:"thanosImage"`
+	ThanosImage string `json:"thanosImage"`
 	// Thanos Image version
-	Version string `json:"thanosVersion"`
+	ThanosVersion string `json:"thanosVersion"`
```

Comment:
this should be under each component, not one version for all, versions should be individually selectable